src/grids.cpp

```diff
@@ -1790,6 +1838,11 @@ std::unique_ptr<NTv2GridSet> NTv2GridSet::open(PJ_CONTEXT *ctx,
         pj_ctx_set_errno(ctx, PJD_ERR_FAILED_TO_LOAD_GRID);
         return nullptr;
     }
+    if (memcmp(header + 56, "SECONDS", 7) != 0) {
```

Comment:
This looks like a magic number of some kind. A comment about what it is or why it is here would be helpful.

Reply:
symbolic name added